Detectors/EMCAL/base/include/EMCALBase/Point.h

```diff
@@ -0,0 +1,98 @@
+#ifndef ALICEO2_EMCAL_POINT_H
+#define ALICEO2_EMCAL_POINT_H
+
+#include <FairMCPoint.h>
+
+namespace AliceO2 {
+  namespace EMCAL {
+    
+    /// \class Point
+    /// \brief EMCAL simulation point information
+    class Point : public FairMCPoint {
+    public:
+      
+      /// \brief Default constructor
+      Point();
+      
+      /// \brief Point constructor
+      ///
+      /// Fully defining information of the EMCAL point (position,
+      /// momentum, energy, track, ...)
+      ///
+      /// \param shunt
+      /// \param primary Number of primary particle
+      /// \param trackID Index of the track
+      /// \param parentID ID of the parent primary entering the EMCAL
+      /// \param detID ID of the detector segment
+      /// \param initialEnergy Energy of the primary particle enering the EMCAL
+      /// \param pos Position vector of the point
+      /// \param mom Momentum vector for the particle at the point
+      /// \param tof Time of the hit
+      /// \param length Length of the segment
+      /// \param EventId Event ID
+      Point(Int_t shunt, Int_t primary, Int_t trackID, Int_t parentID, Int_t detID, Int_t initialEnergy, TVector3 pos, TVector3 mom,
+            Double_t tof, Double_t eLoss, UInt_t EventId=0);
```

Comment:
Concerning mInitialEnergy it is converted from AliEMCALHit. It was probably used to study the evolution of EMCAL showers during development and has maybe no other purpose, but I need a closer check. Unless it would do any harm I would prefer to keep it.